server/routes/reviewPlan/reviewCheckYourAnswersController.test.ts

```diff
@@ -49,7 +49,7 @@ describe('ReviewCheckYourAnswersController', () => {
       await controller.getReviewCheckYourAnswersView(req, res, next)
 
       // Then
-      expect(res.render).toHaveBeenCalledWith('pages/reviewPlan/checkYourAnswers', expectedViewData)
+      expect(res.render).toHaveBeenCalledWith('pages/reviewPlan/checkYourAnswers/index', expectedViewData)
```

Comment:
I should have spied this is an earlier PR, but I've moved the nunjucks template into a folder called `checkYourAnswers`, rather than it being called `checkYourAnswers.njk` (so it's consistent with the other Review based templates 👍 )